src/main/java/examples/CoreExamples.java

```diff
@@ -98,7 +98,7 @@ public void workerExecutor3(Vertx vertx) {
     int poolSize = 10;
 
     // 2 minutes
-    long maxExecuteTime = 120000;
+    long maxExecuteTime = 120000000000L;
```

Comment:
I think we should change this example to call 2 minutes with `TimeUnit.SECONDS` instead